src/librustc_metadata/creader.rs

```diff
@@ -472,6 +465,41 @@ impl<'a> CrateLoader<'a> {
         }
     }
 
+    fn dlsym_proc_macros(&self,
```

Comment:
Nit: could you move this code to its old place (again, hard to see the difference).